scene/main/node.h

```diff
@@ -252,7 +252,7 @@ class Node : public Object {
 		// you can make your own, but don't use the same numbers as other notifications in other nodes
 		NOTIFICATION_ENTER_TREE = 10,
 		NOTIFICATION_EXIT_TREE = 11,
-		NOTIFICATION_MOVED_IN_PARENT = 12,
+		NOTIFICATION_CHILDREN_CHANGED = 12,
```

Comment:
Might be better to use a new index for it and keep it reserved. It's breaking source compatibility anyway, but this way it at least won't send notification to old extensions (which won't expect it when nodes are added/removed).

Reply:
Makes sense. @akien-mga what do you think?

Reply:
Yeah that makes sense. Then I'd keep `NOTIFICATION_MOVED_IN_PARENT` as is so we don't break the API, and just write in its documentation that it's deprecated and never emitted by the engine.

Reply:
Alright